src/script/interpreter.cpp

```diff
@@ -205,7 +207,8 @@ bool CheckSignatureEncoding(const vector<unsigned char> &vchSig, unsigned int fl
     } else if ((flags & SCRIPT_VERIFY_LOW_S) != 0 && !IsLowDERSignature(vchSig, serror)) {
         // serror is set
         return false;
-    } else if ((flags & SCRIPT_VERIFY_STRICTENC) != 0 && !IsDefinedHashtypeSignature(vchSig)) {
+    } else if ((flags & SCRIPT_VERIFY_STRICTENC) != 0 
```

Comment:
I think it's fine, it means that transactions which have `SIGHASH_2X_REPLAY_PROTECT` become standard so that they'll be propagated on 2x. But these transactions are inherently replay protected by the "set bit 8" rule below (the signature itself can be valid on only one chain), there would be no risk if they became standard or were otherwise communicated directly to miners on the original chain.